src/EditorControls.js

```diff
@@ -40,8 +40,6 @@ class EditorControls extends Component {
       data: gd.data,
       dataSources: this.props.dataSources,
       dataSourceOptions: this.props.dataSourceOptions,
-      dataSourceValueRenderer: this.props.dataSourceValueRenderer,
```

Comment:
euh...why are we removing this?
this is actually a prop that's needed for one of our clients..

Reply:
we can't just delete props we used to have on our components, streambed is not the only RCE user

Reply:
Oh, we have to talk...

Reply:
Actually, as long as we don't hit 1.0, we can change the API in any way we want!

Reply:
well.. no im not sure.. let's talk